package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "npm": "^6.1.0",
```

Comment:
Tohle je zbytečný - npm bys měl mít nainstalovaný globálně, odtud bych to oddělal

Reply:
Opraveno.